src/e2e-test/java/io/cdap/plugin/aribasource/stepsdesign/DesignTimeSteps.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.aribasource.stepsdesign;
+import io.cdap.plugin.aribasource.actions.AribaSourcePropertiesPageActions;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import java.io.IOException;
+
+/**
+ * Represents - Ariba - Source plugin - Properties page - Steps.
+ */
+public class DesignTimeSteps {
```

Comment:
Rename class, these steps are not design time.

Reply:
Renamed